03-wordemb-pytorch/wordemb-skip.py

```diff
@@ -3,29 +3,23 @@
 import time
 import random
 import torch
-import torch.nn.functional as F
 
 
-class Skip(torch.nn.Module):
+class WordEmbSkip(torch.nn.Module):
     def __init__(self, nwords, emb_size):
-        super(Skip, self).__init__()
+        super(WordEmbSkip, self).__init__()
 
         """ word embeddings """
         self.word_embedding = torch.nn.Embedding(nwords, emb_size)
         # uniform initialization
         torch.nn.init.uniform_(self.word_embedding.weight, -0.25, 0.25)
         """ context embeddings"""
-        self.context_embedding = torch.nn.Embedding(nwords, emb_size)
-        # uniform initialization
```

Comment:
Is there any particular reason why you didn't initialize the context embeddings?